### PR TITLE
feat(editorconfig): adds editorconfig

### PR DESCRIPTION
**Pull request by ghost**
updates code style for psr-2

https://github.com/sitrunlab/learn-kahlan/pull/9#discussion_r117518202

### CODE REVIEW

**Comment by samsonasik** [comment]
please no, coding style is nothing to do with IDE

**Comment by samsonasik** [comment]
I'm planning to use styleci instead https://styleci.io/

**Comment by ghost** [comment]
> I'm planning to use styleci instead https://styleci.io/

Does that catch style issues before they make it to CI?

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 4
+
+[{*.md,*.markdown}]
+trim_trailing_whitespace = false
```
Comment (samsonasik) [inline .editorconfig:20]: please remove this file, I've just applied https://github.com/sitrunlab/learn-kahlan/commit/136f034b048f1ee18d777f8668a2501fc0611b71 for styleCI automatic check. After that, please do rebase https://samsonasik.wordpress.com/2015/09/16/practical-git-4-rebasing-conflicted-task-branch-against-primary-branch/

**Comment by samsonasik** [comment]
I've applied styleCi at https://github.com/sitrunlab/learn-kahlan/commit/136f034b048f1ee18d777f8668a2501fc0611b71 

Check result: https://styleci.io/analyses/qylxNb

**Comment by samsonasik** [comment]
please do git rebase https://samsonasik.wordpress.com/2015/09/16/practical-git-4-rebasing-conflicted-task-branch-against-primary-branch/

File: .editorconfig (hunk 1 above)
Comment (ghost) [reply .editorconfig:20]: Please see comment here: https://github.com/sitrunlab/learn-kahlan/pull/10#issuecomment-302753271

Comment (samsonasik) [reply .editorconfig:20]: this is your own configuration, should not be commited. Or you can place in another location instead of this repo.

Comment (samsonasik) [reply .editorconfig:20]: you can add to your global .gitignore if you want it.

**Comment by ghost** [comment]
I'm closing this pull as it seems you are opposed to EditorConfig. EditorConfig gives contributors a chance to address style issues before they ever happen in CI, helping reduce churn in PRs and cycles spent in CI. Please consider using it in the future. I will address styling concerns in the related pull as mentioned, and once the build is passing as initially suggested.

**Pull request closed without merging**